src/rest.jl

```diff
@@ -81,7 +81,51 @@ function get_access_token(ctx::Context, creds::ClientCredentials)::AccessToken
     opts = (redirect = false, retry_non_idempotent = true)
     rsp = HTTP.request("POST", url, h, body; opts...)
     data = JSON3.read(rsp.body)
-    return AccessToken(data.access_token, data.scope, data.expires_in, now())
+    return AccessToken(data.access_token, data.scope, data.expires_in, datetime2unix(now()))
+end
+
+# cache name
+function _cache_file()
+    return joinpath(homedir(), ".rai", "tokens.json")
+end
+
+# read oauth cache
+function _read_cache()
+    try
+        return copy(JSON3.read(read(_cache_file(), String)))
```

Comment:
```suggestion
        return copy(JSON3.read(read(_cache_file())))
```
JSON3.read can accept bytes directly; no need for an extra copy.